### PR TITLE
Store license as labels to tenants

### PR DESCRIPTION
**Pull request by dzahariev**
**Description**
Store license as labels to tenants, and modify tenant fetcher to process license types for tenants

Changes proposed in this pull request:
- Expose License Type to GQL API 
- Store license type as label to tenant entity
- Adjust tenant fetcher to extract from payload and send license information when process tenants

**Pull Request status**
- [x] Implementation
- [x] Unit tests
- [x] Integration tests
- [x] `chart/compass/values.yaml` is updated 
- [ ] Mocks are regenerated, using the automated script

### CODE REVIEW

File: components/director/internal/tenantfetchersvc/resync/event_page.go [hunk 1]
```diff
@@ -125,7 +125,15 @@ func (ep EventsPage) eventDataToMovedSubaccount(ctx context.Context, eventData [
...
-	subaccountInput, err := constructSubaccountTenant(ctx, jsonPayload, nameResult.String(), subdomain.String(), id, ep)
+	licenseType := gjson.Get(jsonPayload, ep.FieldMapping.LicenseTypeField)
+	var licenseTypeValue *string
+	if licenseType.Exists() && licenseType.Type == gjson.String {
+		licenseTypeValue = &licenseType.Str
+	} else {
+		log.C(ctx).Warnf("Missing or invalid format of licenseType field: %s for tenant with ID: %s", ep.FieldMapping.LicenseTypeField, id)
+	}
+
+	subaccountInput, err := constructSubaccountTenant(ctx, jsonPayload, nameResult.String(), subdomain.String(), id, licenseTypeValue, ep)
```
Comment (kaloyanspiridonov) [inline components/director/internal/tenantfetchersvc/resync/event_page.go:136]: Do the subaccounts have license?

**Review by kaloyanspiridonov — DISMISSED** [review]

**Comment by dzahariev** [comment]
reset-vm
/test pre-main-compass-integration-with-dump

File: components/director/internal/tenantfetchersvc/resync/event_page.go (hunk 1 above)
Comment (dzahariev) [reply components/director/internal/tenantfetchersvc/resync/event_page.go:136]: Not really, they are with the license of its global account.

**Comment by dzahariev** [comment]
The comments are addressed, the PR is ready for review.

**Comment by dzahariev** [comment]
Closed and reopened to force-trigger "license/cla" check that was hang and cannot be triggered otherwise.

**Comment by dzahariev** [comment]
reset-vm
/test pre-main-compass-integration-with-dump

**Review by nyordanoff — APPROVED** [review]

**Pull request merged**